File: CSharp/5kyu/HumanReadableTime.cs
using System;
using System.Linq;

namespace CSharp._5kyu
{
    public static partial class Kata5
    {
        public static string GetReadableTime(int seconds)
        {
            var sec = seconds % 60;
            var minutes = (seconds / 60) % 60;
            var hours = (int)(seconds / 3600);

            return string.Format("{0:d2}:{1:d2}:{2:d2}", hours, minutes, sec);
        }
        public static string GetReadableTime_Kata(int seconds)
        {
            var t = TimeSpan.FromSeconds(seconds);
            return String.Format("{0:00}:{1:00}:{2:00}", (int)t.TotalHours, t.Minutes, t.Seconds);
        }
    }
}

File: JavaScript/5kyu/HumanReadableTime.js
function humanReadable(seconds) {
  var hours = Math.floor(seconds / 3600);
  var minutes = Math.floor(seconds / 60) % 60;
  var sec = Math.floor(seconds % 60);

  return `${hours.zeors()}:${minutes.zeors()}:${sec.zeors()}`;
}

Number.prototype.zeors = function () {
  return this.toString().padStart(2, "0");
};

console.log(humanReadable(0));
